internal/backends/selector/api.rs

```diff
@@ -173,6 +178,40 @@ impl BackendSelector {
         self
     }
 
+    #[i_slint_core_macros::slint_doc]
+    /// Configures this builder to invoke the functions on the supplied application handler whenever winit wakes up the
+    /// event loop.
+    ///
+    /// *Note*: This function is behind the [`unstable-winit-030` feature flag](slint:rust:slint/docs/cargo_features/#backends)
+    ///         and may be removed or changed in future minor releases, as new major Winit releases become available.
+    ///
+    /// See also the [`slint::winit_030`](slint:rust:slint/winit_030) module
+    #[must_use]
+    #[cfg(feature = "unstable-winit-030")]
+    pub fn with_winit_custom_application_handler(
+        mut self,
+        custom_application_handler: impl i_slint_backend_winit::CustomApplicationHandler + 'static,
+    ) -> Self {
+        self.winit_custom_application_handler = Some(Box::new(custom_application_handler));
+        self
+    }
+
+    #[i_slint_core_macros::slint_doc]
+    /// Configures this builder to use the specified libinput event filter hook when the LinuxKMS backend
+    /// is selected.
```

Comment:
The docs could explain what happens to the returned event.

It says it is a "filter" but how does one filter?  Should it return an Option? Can I generate several event out of it?
Why not `Fn(&input::Event) -> bool` like we do somewhere else.

Reply:
I was mirroring the API of the window attributes hook. But indeed, I can also make it a boolean :)